### DISCUSSION TITLE
Stuck on 1.6.4

### DISCUSSION THREAD

**Discussion by PhasecoreX** [jpanther/congo#99] [Q&A] [answered]
I've followed the instructions to update to 2.0.0 (just run `hugo mod get -u`) but when I do this no messages appear and nothing gets updated. I can see that `go.mod` still has 1.6.4 listed in it. I just started messing with Hugo today actually, and managed to install this theme version 1.6.4 just before the documentation changed over to the 2.0.0 release. Perfect timing!

Is there anything I am doing wrong here? I've tried making a new Hugo project and installing from scratch, I still get this when doing the `hugo server`:
```
go get: added github.com/jpanther/congo v1.6.4
Start building sites … 
hugo v0.92.0+extended linux/amd64 BuildDate=unknown

...
```

Any help would be appreciated!

**Comment by jpanther** [comment, marked as answer]
Thanks, I was missing that you need to append `/v2` to the module path in the `go.mod` and `module.toml` files. If you change the path and try the command again it should find the correct version. The documentation has been updated!

**Reply by mjking** [reply]
I'm having the same issue. I tried adding the v2 to go.mod and module.toml and I'm getting the following:

go get: module github.com/jpanther/congo@upgrade found (v1.6.4), but does not contain package github.com/jpanther/congo/v2

**Reply by mjking** [reply]
Here is my go.mod file 

`module github.com/jpanther/congo/v2

go 1.16`

**Reply by jpanther** [reply]
Can you try using `hugo mod clean` and see if it makes any difference? Perhaps there's something being cached.

**Reply by PhasecoreX** [reply]
My mod file had this line in it:
```
require github.com/jpanther/congo v1.6.4 // indirect
```
The way I got it to work was changing it to the following:
```
require github.com/jpanther/congo/v2 v2 // indirect
```
which then once I ran `hugo mod get -u` it updated to this:
```
require github.com/jpanther/congo/v2 v2.0.1 // indirect
```
So I think I am good now. Not sure if `hugo mod clean` would have fixed this though.

**Comment by mjking** [comment]
hugo mod clean worked for me. thanks!
